tests/test_filters.py

```diff
@@ -81,6 +81,9 @@ def test_default_duplication_filter():
     assert ft.filter(create_record(msg='repeated')) is True
     assert ft.filter(create_record(msg='repeated')) is False
     assert ft.filter(create_record(msg='not repeated')) is True
+    assert ft.filter(create_record(msg='template %s', args=['teste'])) is True
+    assert ft.filter(create_record(msg='template %s', args=['teste'])) is False
+    assert ft.filter(create_record(msg='template %s', args=['teste2'])) is True
```

Comment:
Add another test: `test_duplication_message_template` or something alike

Reply:
@georgeyk done.
github is not showing here.